script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -238,12 +238,12 @@ But he doesn't smell like a bourgeois at all, does he?!"
 [sha:d2dd8340f9831edc527793ef739d937bce5e6789]{
 -- Page 109, Offset 19940.
 -- ……なんなんだ、この状況……。
-...What on earth have I stumbled into here...
+...What on earth have I been caught up in here...
 }
 [sha:55d7f85f6793a819b228c0b9cf91af2240040d8b]{
 -- Page 109, Offset 19941.
 -- とりあえず警察を呼ぶべきか？
-Maybe I should call the cops and let them deal with it?
+Maybe I should just call the cops and let them deal with it?
```

Comment:
`just` gets a lot of use, including _twice_ on the next line. Maybe leave it off here or find something else, unless it really feels necessary for flow